forge-gui/res/cardsfolder/upcoming/tasha_the_witch_queen.txt

```diff
@@ -0,0 +1,13 @@
+Name:Tasha, the Witch Queen
+ManaCost:3 U B
+Types:Legendary Planeswalker Tasha
+Loyalty:4
```

Comment:
`Text:CARDNAME can be your commander.`

Reply:
Thanks, added missing clause after line 4